packages/opds-web-client/src/stylesheets/book.scss

```diff
@@ -1,12 +1,14 @@
-@import "variables";
```

Comment:
How does it work without this?

Reply:
I moved `variables` and `mixins` to the top of `app.scss` so that all the files can have access to it. I _think_ that by importing `variables` into multiple files, it'll be added and compiled multiple times.

Reply:
I think that might break circulation-patron-web as it's currently written. It defines a bunch of its own variables and then imports "app.scss", see https://github.com/NYPL-Simplified/circulation-patron-web/blob/master/src/themes/nypl.scss.

Reply:
I'll test it there to make sure.

Reply:
I tested it and it works...

But I ran into this error: `../opds-web-client/packages/opds-web-client/lib/components/Breadcrumbs.d.ts(12,26): error TS2304: Cannot find name 'Partial'.`

But I updated the TypeScript version in `circulation-patron-web` and it was fine. So I'll send a PR for that too, since it could also use a version bump for `opds-web-client`.